changelogs/fragments/1291-foreman-ansible-inventory-default.yml

```
@@ -0,0 +1,2 @@
+breaking_changes:
+  - Make use_reports_api the default 
```

```suggestion
  - Set use_reports_api default value to true for the inventory plugin
```

Thanks @pipopopo , I made a recommendation to be slightly more specific since this collection includes more than just the inventory plugin